doc/development/documentation.md

```diff
@@ -0,0 +1,168 @@
+# Documentation Site
+
+Flame's documentation is written in **Markdown**. It is then rendered into HTML with the help of
+the [Sphinx] engine and its [MyST] plugin. The rendered files are then manually published to
```

Comment:
`manually` => we have a script :)

Reply:
We havet to manually start the script though, which really should be part of the pipeline once a PR is merged.